### DISCUSSION TITLE
Task list not triggered with link

### DISCUSSION THREAD

**Discussion by nihaals** [community/community#4357] [Projects and Issues]
The `Tracked in` message shown in the tracked issue does not show if you write `- [ ] https://github.com/user/repo/issues/1` instead of `- [ ] #1` but the issue details are embedded. This is a bit confusing as on the issue with the task list, it appears basically the same, the only difference is on the issue being linked from the task list.

**Comment by labrose** [comment]
Hi @nihaals!

Any of the follow syntax should create the tracking relationship:
```
- [ ] #id
- [ ] org/repo#id
- [ ] https://github.com/user_or_org/repo/issues/id
```

In cross repo situations, the link is created if the author has write permissions on both the source and target issue. If you are still not seeing the `Tracked in` message on the tracked issue (issue 1 in your example), could you provide some more details about the surrounding Markdown in your issue?

**Reply by nihaals** [reply]
The repo's public so I can link the actual issue: https://github.com/thunderstore-io/Thunderstore/issues/289

`- [ ] https://github.com/thunderstore-io/Thunderstore/issues/291` definitely doesn't work (I just tried again now) and `- [ ] #281` does. The current content is using the link but I'm assuming you would prefer repro steps.

The content of the tracking issue:
```
Blocked by:
- [ ] https://github.com/thunderstore-io/Thunderstore/issues/291

---

- [ ] Download link for packages
- [ ] Package upload categories
```

**Reply by labrose** [reply]
Thanks for the details! We're investigating.
